### PR TITLE
Implementing Bigtable ColumnRangeFilter.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: gcloud/bigtable/row.py [hunk 1]
```diff
@@ -227,6 +227,80 @@ def to_pb(self):
...
+    flags.
+
+    :type column_family_id: str
+    :param column_family_id: The column family that contains the columns. Must
+                             be of the form ``[_a-zA-Z0-9][-_.a-zA-Z0-9]*``.
+
+    :type start_column: bytes
+    :param start_column: The start of the range of columns. If no value is
+                         used, it is interpreted as the empty string
+                         (inclusive) by the backend.
```
Comment (tseaver) [inline gcloud/bigtable/row.py]: Apply the [same change suggested for `start_value`](https://github.com/GoogleCloudPlatform/gcloud-python/pull/1309#discussion_r48190753) in #1309?

File: gcloud/bigtable/row.py [hunk 2]
```diff
@@ -227,6 +227,80 @@ def to_pb(self):
...
+
+    :type end_column: bytes
+    :param end_column: The end of the range of columns. If no value is used, it
+                       is interpreted as the infinite string (exclusive) by the
+                       backend.
```
Comment (tseaver) [inline gcloud/bigtable/row.py]: Apply the [same change suggested for `end_value`](https://github.com/GoogleCloudPlatform/gcloud-python/pull/1309#discussion_r48190756) in #1309?

File: gcloud/bigtable/row.py [hunk 3]
```diff
@@ -227,6 +227,80 @@ def to_pb(self):
...
+                          included in the range (or excluded).
+    """
+
+    def __init__(self, column_family_id, start_column=None, end_column=None,
+                 inclusive_start=True, inclusive_end=True):
+        self.column_family_id = column_family_id
+        self.start_column = start_column
+        self.end_column = end_column
+        self.inclusive_start = inclusive_start
+        self.inclusive_end = inclusive_end
```
Comment (tseaver) [inline gcloud/bigtable/row.py]: Does this suffer from the case I [wondered about in #1309](https://github.com/GoogleCloudPlatform/gcloud-python/pull/1309#discussion_r48191156)?

**Comment by dhermes** [comment]
@tseaver I gave this the same treatment as in #1309, PTAL.

**Comment by tseaver** [comment]
LGTM

**Pull request merged**